proj1:
  
  joint_state_controller: 
    type: joint_state_controller/JointStateController
    publish_rate: 50
  
  f_controller: 
    type: effort_controllers/JointVelocityController
    joint: front_joint  
    pid : {p: 01.0 , i: 0.0 , d: 0.0}  # Enter your pid values eg. {p: 100.0 , i: 0.0 , d: 0.0} 
    
  # Controller 2
  r_controller:
    type: effort_controllers/JointVelocityController
    joint: back_right_joint  
    pid : {p: 01.0 , i: 0.0 , d: 0.0}  # Enter your pid values eg. {p: 100.0 , i: 0.0 , d: 0.0} 
    
  # Controller 3
  l_controller:
    type: effort_controllers/JointVelocityController
    joint: back_left_joint  
    pid : {p: 01.0 , i: 0.0 , d: 0.0}  # Enter your pid values eg. {p: 100.0 , i: 0.0 , d: 0.0} 
    
 # Controller n


